DateTimeMate.go

```diff
@@ -7,13 +7,13 @@ import (
 	"time"
 
 	"github.com/golang-module/carbon/v2"
+	"github.com/jftuga/parsetime"
 	"github.com/lestrrat-go/strftime"
-	"github.com/tkuchiki/parsetime"
 )
 
 const (
 	ModName    string = "DateTimeMate"
-	ModVersion string = "1.3.1"
+	ModVersion string = "1.3.2"
```

Comment:
It should be considered as a breaking change.

Bumping a major for this would be inappropriate

I would have used this maybe

```suggestion
	ModVersion string = "1.4.0"
```

What do you think?